### PR TITLE
get data from player's server directly

### PR DESCRIPTION
**Pull request by nearnshaw**

### CODE REVIEW

File: _posts/development-guide/2018-02-22-user-data.md [hunk 1]
```diff
@@ -133,43 +133,47 @@ The `getCurrentRealm()` function returns the following information:
...
 import { getUserData } from "@decentraland/Identity"
+import { getCurrentRealm } from "@decentraland/EnvironmentAPI"
 
-const userData = executeTask(async () => {
-  const data = await getUserData()
-  log(data.displayName)
-  return data.displayName
-})
+async fetchPlayerData() {
+    userData = await getUserData()
```
Comment (nchamo) [inline _posts/development-guide/2018-02-22-user-data.md]: I think you need to add `const` or `let` here (and on the line below)

File: _posts/development-guide/2018-02-22-user-data.md [hunk 2]
```diff
@@ -133,43 +133,47 @@ The `getCurrentRealm()` function returns the following information:
...
+    playerRealm = await getCurrentRealm()
 
-fetchPlayerData() {
-    let url = `https://peer.decentraland.org/lambdas/profile/{userData.userId}`.toString()
+    let url = `{playerRealm.domain}/lambdas/profile/{userData.userId}`.toString()
```
Comment (nchamo) [inline _posts/development-guide/2018-02-22-user-data.md:161]: I'm not sure if the domain already contains `https` or not

**Review by nchamo — APPROVED** [review]
Looks good, just a few minor comments

File: _posts/development-guide/2018-02-22-user-data.md (hunk 1 above)
Comment (nearnshaw) [reply _posts/development-guide/2018-02-22-user-data.md]: ahhh well spotted :p
Fixed now

File: _posts/development-guide/2018-02-22-user-data.md (hunk 2 above)
Comment (nearnshaw) [reply _posts/development-guide/2018-02-22-user-data.md:161]: I just did a test and it does include https, so this part should be ok

**Pull request merged**